tron/kubernetes.py

```diff
@@ -135,8 +135,42 @@ def handle_event(self, event: Event) -> None:
         if k8s_type == "running":
             self.started()
         elif k8s_type == "finished":
-            self.exited(0)
+            raw_object = getattr(event, "raw", {})
+            container_statuses = raw_object.get("status", {}).get("container_statuses", [])
+            abnormal_exit = False
+
+            if len(container_statuses) > 1:
+                # shouldn't happen right now, but who knows what future us will do :p
+                self.log.error(
+                    "Got an event for a Pod with multiple containers - not inspecting payload to verify success."
```

Comment:
can we log the actual contents of the container_status so we can investigate this as well?